rice-middleware/ksb/client-impl/pom.xml

```diff
@@ -174,7 +174,7 @@
     </dependency>
     <!-- ASM as an explicit dependency, see KULRICE-6353 -->
     <dependency>
-      <groupId>asm</groupId>
+      <groupId>org.ow2.asm</groupId>
```

Comment:
One thing just in general to check on these, but it would probably be a good idea build the WAR for Rice standalone from these updates, then extract it and check WEB-INF/lib and make sure it doesn't end up with both of the asm jars in it in case that's coming in some other way. In general, I would recommend building the war and just looking at the jars, maybe compare it with the prior war file just to make sure there is nothing weird and no duplicates with different versions.

Also, another kind of weird dependency that you might check out is cglib-nodep. If that ends up in the war for some reason, it has ASM dependencies built into it (and repackaged so as not to conflict). But if ASM 7 is what is required for jdk11 then you might want to check that our cglib is properly up to date too.

Reply:
I updated cglib-nodep to its latest version as well